src/elastic-input.directive.ts

```diff
@@ -66,8 +66,9 @@ export class ElasticInputDirective implements OnInit {
   }
 
   update () {
-    this.mirror.innerText = this.element.nativeElement.value;
+    let domEl = this.element.nativeElement;
```

Comment:
We can set this to `const` as its value is not modified.

Reply:
Totally agree. I wanted to stay consistent. I changed let to const in all of the places